<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1521</width>
    <height>968</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Style général de la fenêtre */
QWidget {
    background-color: #F0F4F8;
    color: #333;
    font-family: 'Segoe UI', 'Roboto', 'Open Sans', sans-serif;
    font-size: 11pt; /* Légèrement réduit pour un aspect plus professionnel */
}

/* Labels */
QLabel {
    color: #1E3A5F; /* Bleu plus profond pour un aspect plus sérieux */
    font-size: 14pt;
    font-weight: 600; /* Moins gras que bold pour plus d’élégance */
}

/* Champs de saisie */
QLineEdit {
    background-color: #FFFFFF;
    border: 2px solid #B0BEC5; /* Gris bleu plus raffiné */
    border-radius: 6px;
    padding: 6px;
    color: #333;
    font-size: 11pt;
}
QLineEdit:focus {
    border: 2px solid #1E88E5; /* Bleu plus vibrant au focus */
    box-shadow: 0px 0px 5px rgba(30, 136, 229, 0.5);
}

/* Boutons */
QPushButton {
    background-color: #1E88E5;
    border: 2px solid #1565C0; /* Bordure plus marquée pour l'élégance */
    border-radius: 8px;
    padding: 8px;
    font-size: 13pt;
    font-weight: 600;
    color: white;
    transition: background 0.3s, box-shadow 0.3s;
}
QPushButton:hover {
    background-color: #1565C0;
    box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.2);
}
QPushButton:pressed {
    background-color: #0D47A1;
    box-shadow: inset 0px 2px 4px rgba(0, 0, 0, 0.3);
}

/* Tableaux */
QTableWidget {
    border: 2px solid #B0BEC5;
    border-radius: 6px;
    gridline-color: #90A4AE;
    background-color: white;
}
QHeaderView::section {
    background-color: #1E88E5;
    color: white;
    padding: 6px;
    border: 1px solid #1565C0;
    font-size: 12pt;
}

/* Barre de défilement */
QScrollBar:vertical {
    background: #E0E7EC;
    width: 12px;
    border-radius: 6px;
}
QScrollBar::handle:vertical {
    background: #1E88E5;
    border-radius: 6px;
}
QScrollBar::handle:vertical:hover {
    background: #1565C0;
}</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1411</width>
      <height>831</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="Ajouter">
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>221</width>
        <height>431</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>201</width>
         <height>401</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>Consultants</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>Projets</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>Clients</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="text">
           <string>Formations</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>Taches</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Resources</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>481</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>AJOUTER CLIENT</string>
      </property>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>441</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>ID Client</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_id"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Nom Complet</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_name"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Nom Entreprise</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_nom_entreprise"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Adresse</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Telephone</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_address"/>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_email"/>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_phone"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="btnAfficher">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>380</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Afficher</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAjouter">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>380</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Desktop/WISEDESK 2.0.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Afficher">
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>221</width>
        <height>461</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>201</width>
         <height>401</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <widget class="QPushButton" name="pushButton_18">
          <property name="text">
           <string>Consultants</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_19">
          <property name="text">
           <string>Projets</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_20">
          <property name="text">
           <string>Clients</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_21">
          <property name="text">
           <string>Formations</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_22">
          <property name="text">
           <string>Taches</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_23">
          <property name="text">
           <string>Resources</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Desktop/WISEDESK 2.0.png</pixmap>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>971</width>
        <height>511</height>
       </rect>
      </property>
      <property name="title">
       <string>CLIENTS</string>
      </property>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>931</width>
         <height>451</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnSupprimer">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>610</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_suppression">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>181</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nbr_consultation">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>30</y>
        <width>181</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAfficher_1">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>610</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnModifier">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>610</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Modifier">
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>221</width>
        <height>461</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>201</width>
         <height>401</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <widget class="QPushButton" name="pushButton_24">
          <property name="text">
           <string>Consultants</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_25">
          <property name="text">
           <string>Projets</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_26">
          <property name="text">
           <string>Clients</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_27">
          <property name="text">
           <string>Formations</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_28">
          <property name="text">
           <string>Taches</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_29">
          <property name="text">
           <string>Resources</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>70</y>
        <width>481</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>AJOUTER CLIENT</string>
      </property>
      <widget class="QWidget" name="formLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>441</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>ID Client</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_id_2"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Nom Complet</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_name_2"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Nom Entreprise</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_nom_entreprise_2"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Adresse</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Telephone</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_address_2"/>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_email_2"/>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_phone_2"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="btnConfirmer">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>370</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Confirmer</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnPrev">
    <property name="geometry">
     <rect>
      <x>1159</x>
      <y>880</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Precedent</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>880</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Suivant</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1521</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
